nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/main/java/org/apache/nifi/csv/CSVRecordSetWriter.java

```diff
@@ -48,6 +54,7 @@
     @Override
     protected List<PropertyDescriptor> getSupportedPropertyDescriptors() {
         final List<PropertyDescriptor> properties = new ArrayList<>(super.getSupportedPropertyDescriptors());
+        properties.add(CSVUtils.EXPLICIT_COLUMNS);
```

Comment:
Hmm. I'll take a stab at defining the new functionality in the `@capabilitydescription` in both `CSVReader` and `CSVRecordSetWriter`. 

I think It's worth documenting, and explaining in `CSVReader` that columns can be renamed by using "Explicit Columns". As well as explaining in `CSVRecordSetWriter` that "Explicit Columns" doesn't rename columns, but does allow for re-ordering, while reminding users that the column names have been defined by whatever RecordReader is being used.